scripts/ide_build.sh

```
@@ -28,8 +28,8 @@ fcDebug="$repo/tests/.fetchcontent-debug"
 stem="$(basename "${src%.*}")"
 
 # clang/libc++ by default, matching cleanbuild.sh's default configuration.
-export CC="/usr/bin/clang-22"
-export CXX="/usr/bin/clang++-22"
+export CC="$(command -v clang)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)